src/App.tsx

```diff
@@ -16,47 +17,99 @@ export const goodsFromServer = [
 ];
 
 export const App: React.FC = () => {
+  const [isReversed, setIsReversed] = useState(false);
+  const [sortBy, setSortBy] = useState('');
+
+  const sortAlphabet = () => {
+    setSortBy('alphabet');
```

Comment:
```suggestion
    setSortBy(SortBy.ALPHABET);
```
і потім його можна так використати для зміни стейту та для кейсів в сортуванні)